async-nats/tests/client_tests.rs

```diff
@@ -290,7 +287,7 @@ mod client {
 
     #[tokio::test]
     async fn token_auth() {
-        let server = nats_server::run_server("tests/configs/token.conf");
+        let server = nats_server::run_server("../async-nats/tests/configs/token.conf");
```

Comment:
Test didn't move, so why the change?

Reply:
Ah right because the crate resolves paths, we should resolve them here instead.

Reply:
yeah, thats why. Removed nats-server resolving the path.